package day58_exception;

public class TryWithInheritance {
    public static void main(String[] args) {

        try {

        }catch (IndexOutOfBoundsException e){     //  IndexOutOfBoundsException obj = new IndexOutOfBoundsException();

        }catch (RuntimeException e){              // RuntimeException obj2 = new IndexOutOfBoundsException();

        }catch (Exception e){                     //  Exception obj3 = new IndexOutOfBoundsException();

        }
    }
}
